src/jackdaw/streams/interop.clj

```diff
@@ -389,6 +400,15 @@
                  (ktable* other-ktable)
                  (value-joiner value-joiner-fn))))
 
+  (suppress
+    [_ {:keys [max-records max-bytes until-time-limit-ms]
+        :as suppress}]
+    (let [cfg (suppress-config suppress)]
+      (println cfg)
```

Comment:
Shall we lose the `println` which I presume was there for debugging purposes.

Reply:
ups...